tests/integ/sagemaker/jumpstart/retrieve_uri/test_inference.py

```diff
@@ -0,0 +1,76 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+
+from tests.integ.sagemaker.jumpstart.retrieve_uri.inference import (
+    EndpointInvoker,
+    InferenceJobLauncher,
+)
+from sagemaker import image_uris
+from sagemaker import script_uris
+from sagemaker import model_uris
+
+from tests.integ.sagemaker.jumpstart.retrieve_uri.constants import InferenceTabularDataname
+
+from tests.integ.sagemaker.jumpstart.retrieve_uri.utils import (
+    download_inference_assets,
+    get_tabular_data,
+)
+
+
+def test_jumpstart_inference_retrieve_functions(setup):
+
+    model_id, model_version = "catboost-classification-model", "*"
```

Comment:
to be safe, i would pin the version here. For example, future models might not work on CPU instances.